plugins/reporters/spdx/src/funTest/resources/synthetic-scan-result-expected-output.spdx.json

```diff
@@ -29,6 +29,41 @@
     "licenseDeclared" : "MIT",
     "name" : "proj1",
     "versionInfo" : "0.0.1"
+  }, {
```

Comment:
Commit message: s/render/renders/

Reply:
> cover the inclusion of the main license

Just a small remark: I'm aware of a bug WRT determining the main license and I'm working on a fix, but unfortunately it required bigger refactorings. Just saying that in case you come across "strange exoected results" WRT the main license.